### PR TITLE
File chunking, reset button

### PR DESCRIPTION
**Pull request by ajbowler**
**Changes:**
- An additional server handler to handle a `getgraphchunk` request
  - Heavy credit to @rwhite226 for getting the graph data chunking logic on the backend working, and chunking for nodes and edges runs concurrently, too!
- Added an option to load the graph data in through chunked sizes, and visualize each chunk iteratively as it comes over the WebSocket connection.
  - Physics had to be reenabled for this, otherwise the data chunks overlap each other on the graph, making it completely unintelligible. Physics allows the nodes to spread out.
  - Total client-side running time, in other words normal rendering vs. chunked rendering, has similar performance, even with the physics enabled. Your mileage may vary based on the machine your browser is running in.
- A reset button that clears all rendered graphs and graph data

No screenshot for chunking, it looks same as always. But here's one for the extra buttons in the Search Panel:

image

### CODE REVIEW

**Comment by cheese-stands-alone** [comment]
+1

**Comment by algmzestrd** [comment]
+1

**Comment by ajbowler** [comment]
Gave everything a test after @rwhite226 's latest changes and it's all good.

@mcsgroi and @winder487 you guys good?

**Comment by mcsgroi** [comment]
+1

File: graphalyzer-server/src/file_handler/os/OSWrapper.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def getOSPathHeader(self):
 			return "c:/"
 		else:
 			self.logger.debug("OS is Linux. Returning path header '/'")
-			return "/"
```
Comment (winder487) [inline graphalyzer-server/src/file_handler/os/OSWrapper.py:52]: My only gripe is that this function essentially doesn't do what the debug log says anymore

Comment (ajbowler) [inline graphalyzer-server/src/file_handler/os/OSWrapper.py:52]: What's wrong with it? @rwhite226 @winder487

Comment (winder487) [inline graphalyzer-server/src/file_handler/os/OSWrapper.py:52]: It's quite minor, but it's no longer returning the path header "/" that it says it is in the Linux logger message. Instead it's returning an empty string.

**Comment by ajbowler** [comment]
There are some bugs that got introduced when I tried merging this with the new code from #24. I'll fix these before merging this.

File: graphalyzer-server/src/file_handler/os/OSWrapper.py (hunk 1 above)
Comment (ajbowler) [inline graphalyzer-server/src/file_handler/os/OSWrapper.py:52]: @rwhite226 thoughts on this?

**Comment by ajbowler** [comment]
I fixed the merge conflict and the bugs that ensued, after we deal with what @winder487 pointed out it should be good to merge.

File: graphalyzer-server/src/file_handler/os/OSWrapper.py (hunk 1 above)
Comment (cheese-stands-alone) [inline graphalyzer-server/src/file_handler/os/OSWrapper.py:52]: The way it was before it was trying to write logs at /neo4j_file_handling which requires root permission. The way it works not is it puts it wherever you currently are. For example if you are at /home/yourname/somefolder/Graphalizer/graphalizer-server/ and run the server the logs will be there. This is better than what it was but we will probably need to change it later to allow a specific configurable location.
**tl;dr leave it and fix it later.**

**Comment by cheese-stands-alone** [comment]
Are you content now @winder487

**Comment by winder487** [comment]
Good enough for me. +1
